cypress/fixtures/handleLogin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "status": "success",
+  "data": {
+    "id": 1,
+    "email": "user@mail.com",
+    "provider": "email",
+    "uid": "user@mail.com",
+    "allow_password_change": false
+  }
+}
```

Comment:
There's data sent from the API not reflected here. You should talk to the API team 😏

Reply:
fixed